packages/terra-docs/src/Notice.module.scss

```diff
@@ -0,0 +1,34 @@
+// Themes
+@import './clinical-lowlight-theme/Notice.module';
+@import './orion-fusion-theme/Notice.module';
+
+:local {
+  .deprecation {
+    background-color: var(--terra-docs-notice-deprecation-background-color, #fff9e7);
+    border-color: var(--terra-docs-notice-deprecation-border-color, #ffc413);
+    color: var(--terra-docs-notice-deprecation-color, #1c1f21);
+  }
+
+  .maintenance {
+    background-color: var(--terra-docs-notice-maintenance-background-color, #e9f5e0);
+    border-color: var(--terra-docs-notice-maintenance-border-color, #a5d784);
```

Comment:
Did UX give us green as the background color?

Reply:
nope, selecting green was random, @jeremyfuksa cleaned up the colors for me though.

Reply:
I see above they did. 👍